### PR TITLE
Branch User Guide for Delete workout

### PR DESCRIPTION
**Pull request by Musfirahe0556596**
Added user guide for delete workout feature

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -47,6 +50,51 @@ A new workout of carrying out Russian twists 50 times will be created and added
 
 ---
 
+### Delete a Workout: `workout /delete`
+Remove an existing workout from the workout list.
```
Comment (alanlowzies) [inline docs/UserGuide.md:54]: Consider changing 'remove' to 'removes' for consistency.

File: docs/_config.yml [hunk 2]
```diff
@@ -1 +1 @@
-theme: jekyll-theme-cayman
```
Comment (alanlowzies) [inline docs/_config.yml:1]: Keep this line as it determines the theme of the website.

**Review by alanlowzies — COMMENTED** [review]
Looks good. Just some changes for you to consider.

**Comment by TianaiYan** [comment]
Looks great!

**Pull request merged**